robots/text.js

```diff
@@ -21,9 +21,11 @@ async function robot(content) {
   async function fetchContentFromWikipedia(content) {
     const algorithmiaAuthenticated = algorithmia(algorithmiaApiKey)
     const wikipediaAlgorithm = algorithmiaAuthenticated.algo('web/WikipediaParser/0.1.2')
-    const wikipediaResponse = await wikipediaAlgorithm.pipe(content.searchTerm)
-    const wikipediaContent = wikipediaResponse.get()
-
+    const wikipediaResponde = await wikipediaAlgorithm.pipe({
```

Comment:
Modificar o wikipediaResponde para wikipediaResponse
```suggestion
    const wikipediaResponse = await wikipediaAlgorithm.pipe({
```